src/Compilers/Core/Desktop/CodeAnalysisDesktopResources.resx

```diff
@@ -132,4 +132,10 @@
   <data name="MissingKeepAlive" xml:space="preserve">
     <value>Missing argument for '/keepalive' option.</value>
   </data>
+  <data name="RuleSetBadAttributeValue" xml:space="preserve">
+    <value>The attribute {0} has an invalid value of {1}.</value>
+  </data>
+  <data name="RuleSetMissingAttribute" xml:space="preserve">
+    <value>The element {0} is missing an attribute named {1}.</value>
+  </data>
```

Comment:
Is it possible to add information about location of the error? The document might have a lot of elements/attributes with the same name.